images/dind/Dockerfile

```diff
@@ -72,6 +78,8 @@ RUN systemctl enable docker.service
 RUN echo "DOCKER_STORAGE_OPTIONS=--storage-driver vfs" >\
  /etc/sysconfig/docker-storage
 
+COPY openshift-dind-lib.sh /usr/local/bin/
```

Comment:
@marun the master needs it too, which is why it's now shared in the base image between master and node.  Do you mean just have both master and node copy the file in?  I wasn't sure how to do that if the file wasn't actually in the current directory with a Dockerfile.  Can I do "COPY ../foobar"?